src/main/java/com/iota/iri/service/API.java

```diff
@@ -268,7 +264,7 @@ private AbstractResponse process(final String requestString, InetSocketAddress s
                         numWalks = minRandomWalks;
                     }
                     try {
-                        final Hash[] tips = getTransactionToApproveStatement(depth, reference, numWalks);
+                        final Hash[] tips = getTransactionToApproveStatement(depth, reference, referenceTips, count, numWalks);
```

Comment:
The return value (dto) does not support returning more than 2 tips. (explicitly gets `tips[0]` , `tips[1]` )

Reply:
good call - but since the dto defines specifically `trunk` and `branch`, how could we modify this to work to our advantage?

I'd be most in favor of modifying the return value to just return `String[] tips` altogether, and at the very least when `count` is provided. But at that point, it begs the question - should this be a new API call (with the count variable)?